nativelib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -72,6 +75,9 @@ object pthread {
                             stackaddr: Ptr[Byte],
                             stacksize: CSize): CInt = extern
 
+  def pthread_attr_setstackaddr(attr: Ptr[pthread_attr_t],
+                                stackaddr: Ptr[Byte]): CInt = extern
+
   def pthread_attr_setstacksize(attr: Ptr[pthread_attr_t],
                                 stacksize: CSize): CInt = extern
 
```

Comment:
[here](http://pubs.opengroup.org/onlinepubs/007908799/xsh/threads.html)(SUSv2) is the minimum you'd need for getting threads going. Anything above that which is also missing from a target platform is probably excess baggage at this point.